### PR TITLE
Remove the semi-colon in the API

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Use JSTransformer Boilerplate to create and update transformers.
 ## API
 
 ```js
-var foo = require('jstransformer')(require('jstransformer-foo'));
+var foo = require('jstransformer')(require('jstransformer-foo'))
 
 foo.render('blah').body
```
Comment (hemanth) [inline README.md]: Rather `foo.render('blah').body;` ;) ?

Comment (RobLoach) [inline README.md]: You'd rather have the semi-colon? I'm okay with either :+1:

Comment (calebeby) [inline README.md]: I like it better without

Comment (hemanth) [inline README.md]: image

**Pull request merged**